app/models/application_record.rb

```diff
@@ -1,3 +1,9 @@
+# frozen_string_literal: true
+
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  # scoping
+  scope :sorted, -> { order('name ASC') }
```

Comment:
No need to change this, a few things to think about:
in a different app this would be a bit harsh - if might have  a model without a `name` column.
you can also define such a thing in a [concern](https://api.rubyonrails.org/v5.2.1/classes/ActiveSupport/Concern.html) (not a classic use of one but still worth reading about)